web/src/left_sidebar_navigation_area.js

```diff
@@ -46,6 +46,7 @@ export function deselect_top_left_corner_items() {
     remove($(".top_left_starred_messages"));
     remove($(".top_left_mentions"));
     remove($(".top_left_recent_topics"));
+    remove($(".top_left_inbox"));
```

Comment:
This `remove` function has a very odd name; it's also probably not ideal to do this for each individual selector rather than a single call; so maybe we should just make a note to rewrite this aspect of this module.

Reply:
Added a comment.

Reply:
Should we open a follow-up issue for this change so we can click resolve here?

Reply:
Opened #26686